README.md

```diff
@@ -26,281 +26,31 @@ docker pull norsys/rothenberg && wget -O - https://github.com/norsys/rothenberg/
-But if the problem always exists, open an issue to help us to improve `rothenberg`.  
-If you open an issue, describe your problem precisely and give maximum information about your environment:
-
-- Operating system ;
-- `docker` version, do `docker --version` to obtain it ;
-- `docker-compose` version, do `docker-compose --version` to obtain it ;
-- `make version`, do `make --version` to obtain it ;
-- Give output of `docker ps -a` ;
-- Give output of `docker network ls` ;
-
-Moreover, if you encounter a problem during installation, reexecute it with `(export WITH_DEBUG=yes; wget -O - https://github.com/norsys/rothenberg/raw/master/install.sh | sh)`, and add the output to your issue.  
-And if you encounter a problem during a `make` command execution, reexecute it with `make <YOUR TARGET HERE> WITH_DEBUG=yes`, and add the output to your issue.  
-Finaly, you can join the channel `##rothenberg` on the IRC network `freenode`:
-
-1. Download an IRC client ;
-2. Configure it with your personal information ;
-3. Use it to [connect to the `freenode` network](https://freenode.net/kb/answer/chat) ;
-4. Do `/j ##rothenberg` ;
-5. Say `Hello!`.
+ - Docker and docker-compose configuration
+ - Docker images management
+ - Default PHP configuration for `CLI` and `FPM`
+ - Helpers to hide `docker` and `docker-compose`
+ - Atoum configuration
+ - Automated `nginx` virtual host management
+ - Networking
+ - Environment management
+ - Private PHP package management
+ - Default `Makefile`
+ - Project Management
+ - Git configuration
+ - Assets watcher
+
+All features are detailled in the [Features](doc/Features.md) section
+
+## Installation and Usage
+All you need to know about how to install is available in the [Usage](doc/Usage.md) file
+
+## Update / TroubleShooting
+Got a problem? What to update ?
+Check [TroubleShooting](doc/Troubleshooting.md) guide !
 
 ## Contributing
-
-### About workflow
-
-We're using pull request to introduce new features and bug fixes.  
-Please, try to be explicit in your commit messages:
-
-1. Explain why the change was made ;
-2. Explain technical implementation (you can provide links to any relevant tickets, articles or other resources).
-
-You can use the following template:
-
-```
-# If applied, this commit will...
-
-# Explain why this change is being made
-
-# Provide links to any relevant tickets, articles or other resources
-```
-
-To use it, just put it in a text file in (for example) your home and define it as a template:
-
-```
-# git config --global commit.template ~/.git_commit_template.txt
-```
-
-### About testing
-
-There are some `make` targets to test `Rothenberg`, especially install and update of bundle and application.  
-To run them, just do `make tests`.  
-Please, do not omit to update tests before implemeting new feature or doing a bug fix.  
-To update tests, just update the content of the `references` directory.
+If you want to contribute, please refer to the [Contribution](doc/Contributing.md) section
```

Comment:
`Contributing.md` must be in root folder ;)

Reply:
Fixed !